README.md

```diff
@@ -140,6 +140,19 @@ $ clang \
 -o example example.c
 ```
 
+The builds of the toolchain come packaged with two config files, Omax.cfg and OmaxLTO.cfg.
+When used, these config files enable several build optimisation flags. OmaxLTO.cfg enables LTO
```

Comment:
Would be also nice to mention that the flags used are internal ones, any advice where to look to explanation of what these options are doing?

Reply:
I've updated and provided a link to clang documentation in my newest patch.